src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/vuex/RunReportPageTests.kt

```diff
@@ -86,6 +86,12 @@ class RunReportPageTests : SeleniumTest()
 
         val gitBranch2 = driver.findElement(By.id("git-branch"))
         assertThat(gitBranch2.getAttribute("value")).isEqualTo("other")
+
+        // clicking Refresh Git button resets git branch back to master
+        driver.findElement(By.id("git-refresh-btn")).click()
```

Comment:
If you're going to include these checks in an existing test, would be good to indicate their inclusion in the test name

Reply:
I could but I was thinking in the long-term, this test might become much longer as we were favouring longer multi-part selenium tests, rather than lots of short ones. The convention then is to explain what the test is doing inline like above, rather than try to load everything into the test name.

Reply:
Fair enough, good to have the comment inline